### PR TITLE
Player database retrieval functions

### PR DESCRIPTION
**Pull request by lattai**
Created helper funcitons in PlayerDatabase to get player entity info.

### CODE REVIEW

File: src/main/java/com/google/sps/data/PlayerDatabase.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    String displayName = player.getDisplayName();
+    String email = player.getEmail();
+    String imageID = player.getImageID();
+    String currentPageID = player.getCurrentPageID();
+    Entity entity = new Entity(ENTITY_QUERY_STRING);
+    entity.setProperty(DISPLAY_NAME_QUERY_STRING, displayName);
+    entity.setProperty(EMAIL_QUERY_STRING, email);
+    entity.setProperty(ID_QUERY_STRING, id);
+    entity.setProperty(IMAGE_ID_QUERY_STRING, imageID);
+    entity.setProperty(CURRENT_PAGE_ID_QUERY_STRING, currentPageID);
```
Comment (DirtGrubDylan) [inline src/main/java/com/google/sps/data/PlayerDatabase.java:46]: Can we make a method to create an entity from a Player?

File: src/main/java/com/google/sps/data/PlayerDatabase.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    datastore.put(entity);
+  }
+
+  // get entity
+  private Entity getCurrentPlayerEntity() {
+    if (UserServiceFactory.getUserService().getCurrentUser() != null) {
```
Comment (DirtGrubDylan) [inline src/main/java/com/google/sps/data/PlayerDatabase.java:52]: Nit: Can we just do:
```
if (UserServiceFactory.getUserService().getCurrentUser() == null) {
  return null;
}

User currentUser = UserServiceFactory.getUserService().getCurrentUser();
String email = currentUser.getEmail();
DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
.
.
.
```

**Comment by lattai** [comment]
When switching branches to make the changes to this PR, I had committed the files I was working on from the other branch with all the blobstore related files. This brought those files onto this branch, which required them to be checked by the linter, and so now instead of the original 1 file being changed in this PR, its 13 files. I'm really sorry about this, definitely my error. 

That being said, these files are just about ready to be merged, so merging them all together seems like a good possibility.

File: pom.xml [hunk 3]
```diff
@@ -69,21 +69,26 @@
...
+        <deploy.projectId>cs-career-step-2020</deploy.projectId>
+        <deploy.version>1</deploy.version>
         </configuration>
-      </plugin>
+    </plugin>
     </plugins>
-  </build>
-</project>
+</build>
+</project>
```
Comment (DirtGrubDylan) [inline pom.xml:94]: New line

File: src/main/java/com/google/sps/data/PlayerDatabase.java (hunk 1 above)
Comment (DirtGrubDylan) [reply src/main/java/com/google/sps/data/PlayerDatabase.java:46]: Can you use your new method here?

File: src/main/java/com/google/sps/servlets/UploadImageServlet.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/servlets/UploadImageServlet.java:1]: This could be in a separate PR too

File: src/main/java/com/google/sps/servlets/UserAuthServlet.java [hunk 5]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2019 Google LLC
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/servlets/UserAuthServlet.java:1]: Do we need this change in this PR?

File: src/test/java/com/google/sps/PlayerTest.java [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  public UserAuthServlet newUserAuthServlet() {
+    DatastoreService localDatastore = DatastoreServiceFactory.getDatastoreService();
+    playerDatabase = new PlayerDatabase(localDatastore);
+    playerDatabase.addPlayerToDatabase(player, TEST_ID_1);
+    UserAuthServlet userAuthServlet = new UserAuthServlet();
+    return userAuthServlet;
+  }
+
+  @Test
+  public void createsNewEntity() throws IOException {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/PlayerTest.java:62]: Add unit tests for every public method in the UserAuthServlet.

File: src/test/java/com/google/sps/PlayerTest.java [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+// Responsible for testing the UserAuthServlet to make sure that players can log in and log out
+// of the webpage.
+@RunWith(JUnit4.class)
+public final class PlayerTest {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/PlayerTest.java:30]: Should this be called UserAuthServletTest?

File: src/test/java/com/google/sps/UploadImageTest.java [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+// Responsible for testing the UploadImage and ImageHandler Servlets to make sure
+// They create accurate links.
+@RunWith(JUnit4.class)
+public final class UploadImageTest {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/UploadImageTest.java:27]: UploadImageServletTest

File: src/test/java/com/google/sps/UploadImageTest.java [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+  // Creating local blobstore service
+  public UploadImageServlet newUploadImageServlet() {
+    BlobstoreService localBlobstore = BlobstoreServiceFactory.getBlobstoreService();
+    UploadImageServlet uploadImageServlet = new UploadImageServlet();
+    return uploadImageServlet;
+  }
+
+  @Test
+  public void uploadsProperly() throws IOException {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/UploadImageTest.java:50]: This should be named for the method it's testing and the thing it's testing, so "doGet_uploadsImage"
